source/tutorial/create-system-dsn.txt

```diff
@@ -12,41 +12,161 @@ Create a System DSN
+           deployment to the same host that serves the |bi-short|. 
+         - If your keytab does not use the default name 
+           (``krb5.keytab``), you must set the 
+           `KRB5_KTNAME <https://web.mit.edu/kerberos/krb5-latest/doc/basic/keytab_def.html>`_ system variable.
+
+           .. note::
+
+              The Kerberos implementation on macOS, `Heimdal`_, does
+              not support the ``default_keytab_name`` configuration 
+              setting, like MIT Kerberos does. You must set the 
+              ``KRB5_KTNAME`` system variable. 
+
+         - If your Kerberos configuration file does not use the 
+           default name (``krb5.conf``), you must set the
+           `KRB5_CONFIG <https://web.mit.edu/kerberos/krb5-devel/doc/admin/conf_files/krb5_conf.html>`_ 
+           system variable. (Heimdal also supports 
+           `this system variable <https://www.h5l.org/manual/HEAD/info/heimdal/Configuration-file.html#Configuration-file>`_.)
+
+         - If you are using a ``mongosqld`` configuration file, you
+           must add the following settings.
+
+           **MongoDB BI Connector Authentication**
+
+           - :setting:`security.enabled` ``: true``
+           - :setting:`security.defaultMechanism` ``: "GSSAPI"``
+           - :setting:`security.defaultSource` ``: "$external"``
+           - :setting:`security.gssapi.hostname` 
+             ``: <host running mongosqld>``
+           - :setting:`security.gssapi.serviceName` 
+             ``: <name of mongosqld service>``
+             (:setting:`security.gssapi.hostname` should match the value in :setting:`serviceName`.)
+
+           .. example::
+
+              .. code-block:: yaml
+   
+                 security
+                   enabled : true
+                   defaultMechanism : "GSSAPI"
+                   defaultSource : "$external"           
+                   gssapi
+                     hostname : <host running mongosqld>
+                     serviceName : <name of mongosqld service>
+
+           **MongoDB Authentication**
+
+           - :setting:`mongodb.net.uri` 
+             ``: "mongodb://<hostname and port of mongodb database>/"``
+           - :setting:`mongodb.net.auth.username` 
+             ``: <database username>``
+           - :setting:`mongodb.net.auth.password` 
+             ``: <database password>``
+           - :setting:`mongodb.net.auth.source` ``: "$external"``
+           - :setting:`mongodb.net.auth.mechanism` ``: "GSSAPI"``
+
+           .. example::
+
+              .. code-block:: yaml
+   
+                 mongodb:
+                  net:
+                    uri: "mongodb://<hostname and port of mongodb database>/"
+                    auth:
+                      username: <database username>
+                      password: <database password>
+                      source: "$external"
+                      mechanism: "GSSAPI"
+
+         .. topic:: Testing |bi-short| with Kerberos on ``localhost``
```

Comment:
I don't think I'm a fan of the topic thing.  Can we try a note or important here instead?

Reply:
It's a sidebar. This localhost only applies in certain situations, but it needs a stronger heading than what we could give with an admonition. Again, I'm chalking this up to CSS.